src/pyAMReX.cpp

```diff
@@ -114,6 +115,9 @@ PYBIND11_MODULE(amrex_3d_pybind, m) {
     init_ParticleContainer(m);
     init_AmrMesh(m);
 
+    // Wrappers around functions, independent of parent to child order
```

Comment:
In fact, these are dependent on the order of types if an argument or return type is used :)

Reply:
ah so should I remove the comment?

Reply:
Maybe this?
```suggestion
    // Wrappers around standalone functions
```